src/styles/main.scss

```
@@ -1,3 +1,15 @@
-body {
-  margin: 0;
-}
+@import './utils/normalize';
+@import './utils/variables';
+@import './utils/functions';
+@import './blocks/calendar';
+
+
+
+
+
+
+
+
+
+
+
```

too many blank lines
```suggestion
@import './blocks/calendar';

```